locales/de/LC_MESSAGES/volto.po

```diff
@@ -1062,11 +1062,11 @@ msgstr "Mehr"
 
 #: components/manage/Contents/ContentsItem
 msgid "Move to bottom of folder"
-msgstr "Zum Anfang des Ordners wechseln"
+msgstr "Ans Ende des Ordners verschieben"
```

Comment:
@ksuess the container could be a page. Shall we just drop the "Ordner"? -> "Ans Ende verschieben"